plugins/hls-class-plugin/test/testdata/AllMethodsRequired.hs

```diff
@@ -0,0 +1,9 @@
+module AllMethodsRequired where
```

Comment:
A corresponding expected file is required, or the test will succeed forever.

Reply:
Those files (`AllMethodsRequired.hs` and `MinimalDefinitionMeet.hs` ) are used to only to [check which code actions are available](https://github.com/batkot/haskell-language-server/blob/a704dfe21329d6d555988e4121f4db2fd8f464cd/plugins/hls-class-plugin/test/Main.hs#L72-L76). There are no golden tests for them, so we don't need any  `*.expected.hs` files.